Gradle test runner does not show line number or stack trace of failed assertion

I have a test that fails only in gradle, and its impossible for me to find out in what line it fails:
```
Failures (1):
  JUnit Jupiter:SeleniumIT:generates a valid video request with a video player()
    MethodSource [className = '....', methodName = 'generates a valid video request with a video player', methodParameterTypes = '']
    => java.lang.AssertionError: 
Expecting actual not to be null
```

junit neither shows a line number nor a stack trace. I think it should show the line number per default. Also i googled and searched the User's guide, and was not able to find a solution.

the console summary does not contain any stacktrace. Here's output from the gradle-consumer demo project, i just added a fail statement to one of the tests:
```
~/Projects/fq/junit5-samples/junit5-gradle-consumer% ./gradlew test
Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details

> Task :junitPlatformTest FAILED

Failures (1):
  JUnit Jupiter:FirstTest:My 1st JUnit 5 test! 😎
    MethodSource [className = 'com.example.project.FirstTest', methodName = 'myFirstTest', methodParameterTypes = 'org.junit.jupiter.api.TestInfo']
    => org.opentest4j.AssertionFailedError: fail

Test run finished after 88 ms
[         6 containers found      ]
[         0 containers skipped    ]
[         6 containers started    ]
[         0 containers aborted    ]
[         6 containers successful ]
[         0 containers failed     ]
[         5 tests found           ]
[         1 tests skipped         ]
[         4 tests started         ]
[         0 tests aborted         ]
[         3 tests successful      ]
[         1 tests failed          ]



FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':junitPlatformTest'.
> Process 'command '/Library/Java/JavaVirtualMachines/jdk-9.0.1.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 7s
4 actionable tasks: 4 executed
```

details: 'flat' does indeed log the stacktrace but thats pretty verbose output just to see the line number of a filed test. 
```
~/Projects/fq/junit5-samples/junit5-gradle-consumer% ./gradlew test

> Task :junitPlatformTest FAILED
Test execution started. Number of static tests: 5
Started:     JUnit Jupiter ([engine:junit-jupiter])
Started:     FirstTest ([engine:junit-jupiter]/[class:com.example.project.FirstTest])
Started:     My 1st JUnit 5 test! 😎 ([engine:junit-jupiter]/[class:com.example.project.FirstTest]/[method:myFirstTest(org.junit.jupiter.api.TestInfo)])
Finished:    My 1st JUnit 5 test! 😎 ([engine:junit-jupiter]/[class:com.example.project.FirstTest]/[method:myFirstTest(org.junit.jupiter.api.TestInfo)])
             => Exception: org.opentest4j.AssertionFailedError: fail
                at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:48)
                at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:36)
                at org.junit.jupiter.api.Assertions.fail(Assertions.java:57)
                at com.example.project.FirstTest.myFirstTest(FirstTest.java:27)
                at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
                at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                at java.base/java.lang.reflect.Method.invoke(Method.java:564)
                at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:389)
                at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:115)
                at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6(TestMethodTestDescriptor.java:167)
                at org.junit.jupiter.engine.execution.ThrowableCollector.execute(ThrowableCollector.java:40)
                at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:163)
                at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:110)
                at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:57)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.lambda$execute$3(HierarchicalTestExecutor.java:83)
                at org.junit.platform.engine.support.hierarchical.SingleTestExecutor.executeSafely(SingleTestExecutor.java:66)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:77)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.lambda$null$2(HierarchicalTestExecutor.java:92)
                at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
                at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
                at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
                at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
                at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
                at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
                at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
                at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
                at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
                at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:430)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.lambda$execute$3(HierarchicalTestExecutor.java:92)
                at org.junit.platform.engine.support.hierarchical.SingleTestExecutor.executeSafely(SingleTestExecutor.java:66)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:77)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.lambda$null$2(HierarchicalTestExecutor.java:92)
                at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
                at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
                at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
                at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
                at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
                at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
                at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
                at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
                at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
                at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:430)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.lambda$execute$3(HierarchicalTestExecutor.java:92)
                at org.junit.platform.engine.support.hierarchical.SingleTestExecutor.executeSafely(SingleTestExecutor.java:66)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:77)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:51)
                at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:43)
                at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:170)
                at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:154)
                at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:90)
                at org.junit.platform.console.tasks.ConsoleTestExecutor.executeTests(ConsoleTestExecutor.java:65)
                at org.junit.platform.console.tasks.ConsoleTestExecutor.lambda$execute$0(ConsoleTestExecutor.java:57)
                at org.junit.platform.console.tasks.CustomContextClassLoaderExecutor.invoke(CustomContextClassLoaderExecutor.java:33)
                at org.junit.platform.console.tasks.ConsoleTestExecutor.execute(ConsoleTestExecutor.java:57)
                at org.junit.platform.console.ConsoleLauncher.executeTests(ConsoleLauncher.java:84)
                at org.junit.platform.console.ConsoleLauncher.execute(ConsoleLauncher.java:74)
                at org.junit.platform.console.ConsoleLauncher.execute(ConsoleLauncher.java:47)
                at org.junit.platform.console.ConsoleLauncher.main(ConsoleLauncher.java:39)
Finished:    FirstTest ([engine:junit-jupiter]/[class:com.example.project.FirstTest])
Started:     OtherTests ([engine:junit-jupiter]/[class:com.example.project.OtherTests])
Started:     testThisThing() ([engine:junit-jupiter]/[class:com.example.project.OtherTests]/[method:testThisThing()])
Finished:    testThisThing() ([engine:junit-jupiter]/[class:com.example.project.OtherTests]/[method:testThisThing()])
Started:     testThisOtherThing() ([engine:junit-jupiter]/[class:com.example.project.OtherTests]/[method:testThisOtherThing()])
Finished:    testThisOtherThing() ([engine:junit-jupiter]/[class:com.example.project.OtherTests]/[method:testThisOtherThing()])
Finished:    OtherTests ([engine:junit-jupiter]/[class:com.example.project.OtherTests])
Started:     SecondTest ([engine:junit-jupiter]/[class:com.example.project.SecondTest])
Skipped:     mySecondTest() ([engine:junit-jupiter]/[class:com.example.project.SecondTest]/[method:mySecondTest()])
             => Reason: void com.example.project.SecondTest.mySecondTest() is @Disabled
Finished:    SecondTest ([engine:junit-jupiter]/[class:com.example.project.SecondTest])
Finished:    JUnit Jupiter ([engine:junit-jupiter])
Started:     JUnit Vintage ([engine:junit-vintage])
Started:     com.example.project.JUnit4Test ([engine:junit-vintage]/[runner:com.example.project.JUnit4Test])
Started:     test ([engine:junit-vintage]/[runner:com.example.project.JUnit4Test]/[test:test(com.example.project.JUnit4Test)])
Finished:    test ([engine:junit-vintage]/[runner:com.example.project.JUnit4Test]/[test:test(com.example.project.JUnit4Test)])
Finished:    com.example.project.JUnit4Test ([engine:junit-vintage]/[runner:com.example.project.JUnit4Test])
Finished:    JUnit Vintage ([engine:junit-vintage])
Test execution finished.

Test run finished after 97 ms
[         6 containers found      ]
[         0 containers skipped    ]
[         6 containers started    ]
[         0 containers aborted    ]
[         6 containers successful ]
[         0 containers failed     ]
[         5 tests found           ]
[         1 tests skipped         ]
[         4 tests started         ]
[         0 tests aborted         ]
[         3 tests successful      ]
[         1 tests failed          ]



FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':junitPlatformTest'.
> Process 'command '/Library/Java/JavaVirtualMachines/jdk-9.0.1.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
4 actionable tasks: 1 executed, 3 up-to-date
```

--- a/documentation/src/docs/asciidoc/release-notes/release-notes-5.1.0-M2.adoc
+++ b/documentation/src/docs/asciidoc/release-notes/release-notes-5.1.0-M2.adoc
@@ -27,6 +27,8 @@ on GitHub.
 * New `SUMMARY` details mode for the `ConsoleLauncher` and Gradle plugin which prints
   the table of success and failure counts at the end of test plan execution. This new
   mode is analogous to the previous behavior of the `NONE` details mode.
+* The summary table of a console launcher run now contains the initial ten stack trace
+  lines to better describe the location of the failure.
 * Non-inherited _composed annotations_ which are meta-annotated with a given `@Inherited`
   annotation are now considered to be implicitly _inherited_ when searching for the given
   meta-annotation within a class hierarchy.

--- a/junit-platform-launcher/src/main/java/org/junit/platform/launcher/listeners/MutableTestExecutionSummary.java
+++ b/junit-platform-launcher/src/main/java/org/junit/platform/launcher/listeners/MutableTestExecutionSummary.java
@@ -16,7 +16,6 @@ import java.io.PrintWriter;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
-import java.util.Optional;
 import java.util.concurrent.atomic.AtomicLong;
 
 import org.junit.platform.launcher.TestIdentifier;
@@ -31,6 +30,7 @@ class MutableTestExecutionSummary implements TestExecutionSummary {
 
 	private static final String TAB = "  ";
 	private static final String DOUBLE_TAB = TAB + TAB;
+	private static final int MAX_STACKTRACE_LINES = 10;
 
 	final AtomicLong containersFound = new AtomicLong();
 	final AtomicLong containersStarted = new AtomicLong();
@@ -140,8 +140,7 @@ class MutableTestExecutionSummary implements TestExecutionSummary {
 	@Override
 	public void printTo(PrintWriter writer) {
 		// @formatter:off
-		writer.println(String.format(
-			"%nTest run finished after %d ms%n"
+		writer.printf("%nTest run finished after %d ms%n"
 
 			+ "[%10d containers found      ]%n"
 			+ "[%10d containers skipped    ]%n"
@@ -155,7 +154,8 @@ class MutableTestExecutionSummary implements TestExecutionSummary {
 			+ "[%10d tests started         ]%n"
 			+ "[%10d tests aborted         ]%n"
 			+ "[%10d tests successful      ]%n"
-			+ "[%10d tests failed          ]%n",
+			+ "[%10d tests failed          ]%n"
+			+ "%n",
 
 			(this.timeFinished - this.timeStarted),
 
@@ -172,7 +172,7 @@ class MutableTestExecutionSummary implements TestExecutionSummary {
 			getTestsAbortedCount(),
 			getTestsSucceededCount(),
 			getTestsFailedCount()
-		));
+		);
 		// @formatter:on
 
 		writer.flush();
@@ -181,12 +181,12 @@ class MutableTestExecutionSummary implements TestExecutionSummary {
 	@Override
 	public void printFailuresTo(PrintWriter writer) {
 		if (getTotalFailureCount() > 0) {
-			writer.println();
-			writer.println(String.format("Failures (%d):", getTotalFailureCount()));
+			writer.printf("%nFailures (%d):%n", getTotalFailureCount());
 			this.failures.forEach(failure -> {
-				writer.println(TAB + describeTest(failure.getTestIdentifier()));
-				failure.getTestIdentifier().getSource().ifPresent(source -> writer.println(DOUBLE_TAB + source));
-				writer.println(String.format("%s=> %s", DOUBLE_TAB, failure.getException()));
+				writer.printf("%s%s%n", TAB, describeTest(failure.getTestIdentifier()));
+				printSource(writer, failure.getTestIdentifier());
+				writer.printf("%s=> %s%n", DOUBLE_TAB, failure.getException());
+				printStackTrace(writer, failure.getException(), MAX_STACKTRACE_LINES);
 			});
 			writer.flush();
 		}
@@ -199,15 +199,27 @@ class MutableTestExecutionSummary implements TestExecutionSummary {
 
 	private String describeTest(TestIdentifier testIdentifier) {
 		List<String> descriptionParts = new ArrayList<>();
-		collectTestDescription(Optional.of(testIdentifier), descriptionParts);
+		collectTestDescription(testIdentifier, descriptionParts);
 		return join(":", descriptionParts);
 	}
 
-	private void collectTestDescription(Optional<TestIdentifier> optionalIdentifier, List<String> descriptionParts) {
-		optionalIdentifier.ifPresent(testIdentifier -> {
-			descriptionParts.add(0, testIdentifier.getDisplayName());
-			collectTestDescription(this.testPlan.getParent(testIdentifier), descriptionParts);
-		});
+	private void collectTestDescription(TestIdentifier identifier, List<String> descriptionParts) {
+		descriptionParts.add(0, identifier.getDisplayName());
+		this.testPlan.getParent(identifier).ifPresent(parent -> collectTestDescription(parent, descriptionParts));
+	}
+
+	private void printSource(PrintWriter writer, TestIdentifier testIdentifier) {
+		testIdentifier.getSource().ifPresent(source -> writer.printf("%s%s%n", DOUBLE_TAB, source));
+	}
+
+	private void printStackTrace(PrintWriter writer, Throwable throwable, int max) {
+		StackTraceElement[] elements = throwable.getStackTrace();
+		for (int i = 0; i < (elements.length > max ? max : elements.length); i++) {
+			writer.printf("%s   %s%n", DOUBLE_TAB, elements[i]);
+		}
+		if (elements.length > max) {
+			writer.printf("%s   [...]%n", DOUBLE_TAB);
+		}
 	}
 
 	private static class DefaultFailure implements Failure {

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-none-ascii.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-none-ascii.out.txt
@@ -6,6 +6,7 @@ Failures (1):
 line
 fail
 message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-none-unicode.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-none-unicode.out.txt
@@ -6,6 +6,7 @@ Failures (1):
 line
 fail
 message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-summary-ascii.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-summary-ascii.out.txt
@@ -6,6 +6,7 @@ Failures (1):
 line
 fail
 message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-summary-unicode.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-summary-unicode.out.txt
@@ -6,6 +6,7 @@ Failures (1):
 line
 fail
 message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-tree-ascii.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-tree-ascii.out.txt
@@ -13,6 +13,7 @@ Failures (1):
 line
 fail
 message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-tree-unicode.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithMultiLineMessage-tree-unicode.out.txt
@@ -13,6 +13,7 @@ Failures (1):
 line
 fail
 message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-none-ascii.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-none-ascii.out.txt
@@ -3,6 +3,7 @@ Failures (1):
   JUnit Jupiter:Fail:failWithSingleLineMessage()
     MethodSource [className = 'org.junit.platform.console.ConsoleDetailsTests$Fail', methodName = 'failWithSingleLineMessage', methodParameterTypes = '']
     => org.opentest4j.AssertionFailedError: single line fail message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-none-unicode.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-none-unicode.out.txt
@@ -3,6 +3,7 @@ Failures (1):
   JUnit Jupiter:Fail:failWithSingleLineMessage()
     MethodSource [className = 'org.junit.platform.console.ConsoleDetailsTests$Fail', methodName = 'failWithSingleLineMessage', methodParameterTypes = '']
     => org.opentest4j.AssertionFailedError: single line fail message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-summary-ascii.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-summary-ascii.out.txt
@@ -3,6 +3,7 @@ Failures (1):
   JUnit Jupiter:Fail:failWithSingleLineMessage()
     MethodSource [className = 'org.junit.platform.console.ConsoleDetailsTests$Fail', methodName = 'failWithSingleLineMessage', methodParameterTypes = '']
     => org.opentest4j.AssertionFailedError: single line fail message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-summary-unicode.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-summary-unicode.out.txt
@@ -3,6 +3,7 @@ Failures (1):
   JUnit Jupiter:Fail:failWithSingleLineMessage()
     MethodSource [className = 'org.junit.platform.console.ConsoleDetailsTests$Fail', methodName = 'failWithSingleLineMessage', methodParameterTypes = '']
     => org.opentest4j.AssertionFailedError: single line fail message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-tree-ascii.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-tree-ascii.out.txt
@@ -7,6 +7,7 @@ Failures (1):
   JUnit Jupiter:Fail:failWithSingleLineMessage()
     MethodSource [className = 'org.junit.platform.console.ConsoleDetailsTests$Fail', methodName = 'failWithSingleLineMessage', methodParameterTypes = '']
     => org.opentest4j.AssertionFailedError: single line fail message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

--- a/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-tree-unicode.out.txt
+++ b/platform-tests/src/test/resources/console/details/fail/Fail-failWithSingleLineMessage-tree-unicode.out.txt
@@ -7,6 +7,7 @@ Failures (1):
   JUnit Jupiter:Fail:failWithSingleLineMessage()
     MethodSource [className = 'org.junit.platform.console.ConsoleDetailsTests$Fail', methodName = 'failWithSingleLineMessage', methodParameterTypes = '']
     => org.opentest4j.AssertionFailedError: single line fail message
+>> STACKTRACE >>
 
 Test run finished after [\d]+ ms
 [         2 containers found      ]

